### PR TITLE
Separate external and internal facing readme sections

### PR DESCRIPTION
**Pull request by Cdok**
# Separate external and internal facing readme sections

 **JIRA**: https://mobify.atlassian.net/browse/WEBDATA-40
 **Linked PRs**: https://github.com/mobify/generator-progressive-web/pull/8 and https://github.com/mobify/progressive-web-sdk/pull/645

## Changes
- Split up readmes into 2 separate files.

## How to test-drive this PR
- Review `DEVELOPING.md` and `README.md` to ensure that `README.md` contains anything a partner would need, and `DEVELOPING.md` contains anything a Mobifyer would need.
- `DEVELOPING.md` will be stripped from the repo when a new project is generated (as per https://github.com/mobify/generator-progressive-web/pull/8 this PR)

### CODE REVIEW

**Comment by Cdok** [comment]
Not sure if the `## Swapping Integration Managers` should be partner-visible, maybe a new section on "Picking a connector" or something along those lines?

File: web/DEVELOPING.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+```
+
+Then navigate back to this directory and run:
+```
+cd ../platform-scaffold/web
+npm link progressive-web-sdk
+npm run dev
+```
+
+## Analyze Bundle Size
```
Comment (jeremywiebe) [inline web/DEVELOPING.md]: This is something that partners will want to be able to know how to do.  Can we move this into the main `README.md`?

File: web/DEVELOPING.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+To visualize bundle script content, run:
+
+```
+npm run analyze-build
+```
+
+## Swapping Integration Managers
+
+Note: This should make it's way into the Tutorial eventually, but this is here for now.
```
Comment (jeremywiebe) [inline web/DEVELOPING.md]: Could you drop in the gist of the content I posted on Slack yesterday?

https://mobify.slack.com/archives/C0DKK1FJS/p1495730437606143

**Comment by jeremywiebe** [comment]
@Cdok With regards to switching connectors... I think that can stay in `DEVELOPING.md`   We'll have full guides on how to set up your project with the connector you need when we ship the IM...

**Pull request merged**